meetings/2024-01-10.md

```diff
@@ -75,33 +90,65 @@ URL: https://virginiabalseiro.github.io/solid-ecosystem-monitor/
+* SC: You might wanna look at PROV-O.
+* FG: We use it. Like 150 other ones? Not many define on how everything is connected. We also have dynamic XXX based on states. Ontology is used operationally, and that something that goes on server side.
+* PAC: Are you aware of RDF-star, could it be an option to help you represent notion of states?
+* FG: Aware of it. Our approach, how do we model how people think. RDF-star is a way to give attributes to a triple. You can kind of do it, which is fine but it doesn't give you XXX entity connect to attribute, and all the states are in the sytem, e.g, colour red is one. We are constantly classifying, grouping, associating this way. We can grab everything coloured red, and when we paint something black, the state is changed. We had to use this for very large datasets. For a high-rise building, in SPARQL it wasn't fast enough. Had to drop graph database. We use APache SQL that scales to that size.
+* PAC: You may want to look at discussions in [RDF-star WG](https://www.w3.org/groups/wg/rdf-star). Not a standard but the way to integrate into RDF 1.2. It'd be different introduce it in a WG.
+* PAC: Re inferences, with the notion of states, can you benefit from standard mechanisms like OWL, or did you build your own?
+* FG: Ours is nested, hierarchical, recursive. A very good ontology that goes down to the object. We built it all. Was difficult at scale to do inference. It was a lot of engineering to make it efficient.
+* VB: Perhaps the access control approach can be in STM.
 
 ### i18n and n11n of resource identifiers
 URL: https://github.com/solid/specification/pull/575
 
+* SC: We need more reviews on the integrity of what's being proposed. Implementation commitments or feedback so that there's something grounded as to why this is being introduced. So far disucssion is good. There are implications on related specifications so good to give more time or attention, security related, access controls, how it indluences how WebID is used. Marked as milestone for next release so if we're going to do it make sure we understand.. seems like an easy thing but there's areas where it makes sense and other parts it might be an issue. Implementation experience is important to see what could break. That's important, not just looking at and seeing if it's technically okay. 
+* eP: I recall in one of the Matrix chats: someone was raising requirements for IRI but can't recall where. If someone needs IRI functionality, it would be good to have it. Re RDF canonicalisation. Did you look into that?
+* SC: Good catch. I do remember. It wasn't related. I was thinking of RDF 1.2 concept https://www.w3.org/TR/rdf12-concepts/#section-IRIs actually, not canonicalisation. So that's a correction to what I said before in a meeting.
+* SC: Re: first point, good point but more generally it is important because if you look at deploying things like wikipedia it's important and solid is supposed to not be ??? . 
```

Comment:
```suggestion
* SC: Re: first point, good point, but more generally it is important because if you look at deploying things like Wikipedia, it's important, and Solid is supposed to not be ??? . 
```